.github/workflows/main.yml

```diff
@@ -2,8 +2,6 @@ name: 'CI Multiplatform Build'
 
 on:
   push:
-    branches:
-      - master
```

Comment:
Why was this removed?

Reply:
This makes the CI process run on any branch of the obs repo, right now it only runs on master.